### PR TITLE
Separation of pitch-roll rates.

### PR DESCRIPTION
**Pull request by readerror67**
@trollcop 

Will require changes to configurator for setting but should support legacy applications.

### CODE REVIEW

**Comment by readerror67** [comment]
test this please

**Comment by readerror67** [comment]
@trollcop Please advise.

File: src/config.c [hunk 1]
```diff
@@ -285,7 +285,8 @@ static void resetConf(void)
     cfg.D8[PIDVEL] = 1;
     cfg.rcRate8 = 90;
     cfg.rcExpo8 = 65;
-    cfg.rollPitchRate = 0;
+    cfg.rollPitchRate[0] = 0;
+    cfg.rollPitchRate[1] = 0;
```
Comment (trollcop) [inline src/config.c]: not really needed, array is already zero'd anyway.
old stuff stays, but new zero vars shouldn't be added.

File: src/serial.c [hunk 2]
```diff
@@ -79,6 +79,8 @@
 #define MSP_SET_CONFIG           67     //in message          baseflight-specific settings save
 #define MSP_REBOOT               68     //in message          reboot settings
 #define MSP_BUILDINFO            69     //out message         build date as well as some space for future expansion
+#define MSP_TUNING_EXT           70     //out message         extended tuning params
+#define MSP_SET_TUNING_EXT       71     //in message          extended tuning params
```
Comment (trollcop) [inline src/serial.c]: why did you add tehse if you aren't using them?
